make_sleuth_folders.py

```diff
@@ -1,61 +1,76 @@
+"""This utility creates directories for the Sleuth files for University

```

Comment:
This is the "module docstring". It's always a good idea to have one in case your file is used independently from the rest of your repo and serves as a nice documentation string if you display help for that particular file.